### PR TITLE
[[ Bug 16748 ]] Add datagrid column management to PI

### PR DESCRIPTION
**Pull request by livecodeali**
This change is [Reviewable](https://reviewable.io/reviews/livecode/livecode-ide/1045)

### CODE REVIEW

File: Toolset/palettes/inspector/editors/com.livecode.pi.script.behavior.livecodescript [hunk 1]
```diff
@@ -37,16 +38,23 @@ constant kButtonHeight = 23
...
+   set the width of field 1 of me to tFieldWidth
    set the height of field 1 of me to tFieldHeight
    set the topleft of field 1 of me to the topleft of me
    
-   set the rect of widget 1 of me to 0,0,tFieldHeight,tFieldHeight
-   set the topright of widget 1 of me to the topright of me	
+   if tEnabled then
+      set the rect of widget 1 of me to 0,0,tFieldHeight,tFieldHeight
+      set the topright of widget 1 of me to the topright of me	
+   end if
```
Comment (peter-b) [inline Toolset/palettes/inspector/editors/com.livecode.pi.script.behavior.livecodescript:57]: @livecodeali Are you sure these changes to the script PI editor are supposed to be in this commit?

**Comment by livecodeali** [comment]
@peter-b Updated to clarify the purpose of the script PI editor changes.

**Comment by peter-b** [comment]
Reviewed 1 of 1 files at r1, 1 of 1 files at r2, 2 of 3 files at r3, 5 of 6 files at r4, 1 of 1 files at r5, 1 of 1 files at r6, 2 of 2 files at r7, 1 of 1 files at r8.
Review status: 12 of 13 files reviewed at latest revision, 1 unresolved discussion.

---

_[Toolset/resources/supporting_files/property_definitions/com.livecode.interface.classic.DataGrid.txt, line 18 [r3]](https://reviewable.io:443/reviews/livecode/livecode-ide/1045#-KDYDjX2YLmumHmK-_rQ:-KDYDjX2YLmumHmK-_rR:-64383001) ([raw file](https://github.com/livecode/livecode-ide/blob/d3afe2ba4ee343ba2c6370ac8174ae6dbdde3d1a/Toolset/resources/supporting_files/property_definitions/com.livecode.interface.classic.DataGrid.txt#L18)):_
I think that property labels are now expected to be in sentence case (i.e. "Row Template" -> "Row template")

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/livecode/livecode-ide/1045)_

**Comment by livecodeali** [comment]
_[Toolset/resources/supporting_files/property_definitions/com.livecode.interface.classic.DataGrid.txt, line 18 [r3]](https://reviewable.io:443/reviews/livecode/livecode-ide/1045#-KDYDjX2YLmumHmK-_rQ:-KDbe4qQVmN1Kipm8-hz:-1738440116) ([raw file](https://github.com/livecode/livecode-ide/blob/d3afe2ba4ee343ba2c6370ac8174ae6dbdde3d1a/Toolset/resources/supporting_files/property_definitions/com.livecode.interface.classic.DataGrid.txt#L18)):_
Thanks! I can see myself forgetting that repeatedly.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/livecode/livecode-ide/1045)_

**Comment by peter-b** [comment]
@livecode-vulcan review ok 850a5e6

---

Reviewed 1 of 3 files at r11, 6 of 6 files at r12, 1 of 1 files at r13, 1 of 1 files at r14, 2 of 2 files at r15, 1 of 1 files at r16.
Review status: all files reviewed at latest revision, all discussions resolved.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/livecode/livecode-ide/1045#-:-KDcoqzVWu_PHIlBYg9d:631680120)_

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 850a5e6a72811e2e65364232e83a8dfd536b5ace

**Comment by livecodeali** [comment]
Fixed a trivial merge conflict.
@livecode-vulcan review by @peter-b ok 3e30a16

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 3e30a16613179cc4958c213c3af21ee0467fabf5

**Pull request merged**